client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -137,6 +138,8 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 		setDomain( freeDomainSuggestion );
 	};
 
+	usePresalesChat( 'wpcom' );
```

Comment:
Do we need some sort of `if ( flow === 'hosting' )` guard?

Reply:
I was wondering this, do we need it for only hosting flow? I tried to see where else the plans wrapper was used to see if presale make sense there also, in my opinion it seems like a sensible cross the board thing to do.

Reply:
I'd vote so, yes. If there's push back we can always modify and restrict it to hosting, but I can't see how it'd be harmful.

Reply:
It can be harmful, if this opens up the presales chat to more users than anticipated :) Which sounds like a possibility, since it's the plans component, which I assume is shown in some very high traffic areas. So I'd appreciate making sure the presales chat will only be shown on the pages agreed with the Sales team. As we'll plan staffing around those expected numbers.

Reply:
Thanks for the heads-up, @klimeryk. We will scope it to /hosting 👍